frontend/src/metabase-lib/v1/Question.ts

```diff
@@ -722,7 +722,11 @@ class Question {
       collection_id: this._card.collection_id,
       dataset_query: Lib.toLegacyQuery(query),
       display: this._card.display,
-      parameters: this._card.parameters,
+      ...(_.isEmpty(this._card.parameters)
```

Comment:
This is similar to `parameterValues` check below. Not changing the order of properties to not break comparing with existing hash urls.